src/components/pages/pricing/forecast/forecast.jsx

```diff
@@ -0,0 +1,159 @@
+import LINKS from 'constants/links';
+import { activities, performance, storage } from 'constants/pricing';
+import useWindowSize from 'hooks/use-window-size';
+import ArrowIcon from 'icons/arrow-sm.inline.svg';
+
+import Metrics from './metrics';
+
+const SECTION_MIN_HEIGHT = 760;
+
+const getSelectedIndex = (activeTitle, items) =>
+  items.findIndex(({ title }) => title === activeTitle) + 1;
+
+const Forecast = () => {
+  const sectionRef = useRef();
+  const { width: windowWidth, height: pageHeight } = useWindowSize();
+  const scrollY = useScrollPosition();
+  const [contentRef, isContentInView] = useInView({ triggerOnce: true });
+
+  const [currentSectionIndex, setCurrentSectionIndex] = useState(0);
+
+  const [activeItems, setActiveItems] = useState({
+    activity: activities[0],
+    performance: performance[0],
+    storage: storage[0],
+  });
+
+  const { RiveComponent, rive } = useRive({
+    src: '/animations/pages/pricing/lights.riv',
+    autoplay: false,
+    stateMachines: 'SM',
+    layout: new Layout({
+      fit: Fit.FitWidth,
+      alignment: Alignment.Center,
+    }),
+  });
+
+  const animationStageInput = useStateMachineInput(rive, 'SM', 'stages', 1);
+  const firstSelectInput = useStateMachineInput(rive, 'SM', '1 select', 1);
+  const secondSelectInput = useStateMachineInput(rive, 'SM', '2 select', 1);
+  const thirdSelectInput = useStateMachineInput(rive, 'SM', '3 select', 1);
+
+  useEffect(() => {
+    if (rive && isContentInView) {
+      rive.play();
+    }
+  }, [rive, isContentInView, currentSectionIndex]);
+
+  useEffect(() => {
+    if (!animationStageInput) return;
+
+    if (currentSectionIndex === 3 && animationStageInput.value === 1) {
+      animationStageInput.value = 3;
+    } else {
+      animationStageInput.value = currentSectionIndex + 1;
+    }
+
+    if (!firstSelectInput) return;
+    firstSelectInput.value = getSelectedIndex(activeItems.activity.title, activities);
+
+    if (!secondSelectInput) return;
+    secondSelectInput.value = getSelectedIndex(activeItems.performance.title, performance);
+
+    if (!thirdSelectInput) return;
+    thirdSelectInput.value = getSelectedIndex(activeItems.storage.title, storage);
+  }, [
+    currentSectionIndex,
+    animationStageInput,
+    firstSelectInput,
+    secondSelectInput,
+    thirdSelectInput,
+    activeItems.activity.title,
+    activeItems.performance.title,
+    activeItems.storage.title,
+  ]);
+
+  useEffect(() => {
+    const currentScrollTop = scrollY;
+    const switchPointMultiplier = pageHeight < 975 ? SECTION_MIN_HEIGHT * 1 : pageHeight * 0.9;
+    const switchPoints = [...Array(5)].map(
+      (_, index) => sectionRef.current.offsetTop + 130 + index * switchPointMultiplier - 200
+    );
+
+    switchPoints.forEach((_, index) => {
+      if (currentScrollTop > switchPoints[index] && currentScrollTop < switchPoints[index + 1]) {
+        setCurrentSectionIndex(index);
+      }
+    });
+  }, [pageHeight, scrollY]);
```

Comment:
A lot of magic numbers. It will be great to add some comment with explanation (in that case it will be better than using of CONSTANTS)